easybuild/easyconfigs/v/VASP/VASP-5.4.4-CrayIntel-17.08-cuda-8.0.eb

```diff
@@ -0,0 +1,39 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'MakeCp'
+
+name = 'VASP'
+version = "5.4.4"
+cudaversion = '8.0'
+versionsuffix = '-cuda-%s' % cudaversion
+
+homepage = 'http://www.vasp.at'
+description = """The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles. """
+
+toolchain = {'name': 'CrayIntel', 'version': '17.08'}
+toolchainopts = { 'usempi': True }
+
+patches = [('%(name)s-%(version)s-%(toolchain_name)s-%(toolchain_version)s%(versionsuffix)s.makefile.include', '%(builddir)s/%(namelower)s-%(version)s')]
```

Comment:
can you explain this line ?

Reply:
It's not a patch, I need an additional file `makefile.include` to be copied to the build folder and EasyBuild let you copy a new file with the `patch` syntax. I also need to change its name, and unfortunately I did not succeed to do that within the `patch` instruction, so I had to add the `mv` command in `prebuildopts`: I had tried to do a single `cp` in the `prebuildopts`, but at that stage the EasyBuild script did not find the file, if I had not copied it before.